datalad/downloaders/base.py

```diff
@@ -185,40 +194,67 @@ def access(self, method, url, allow_old_session=True, **kwargs):
                         if not ui.is_interactive:
                             lgr.error(
                                 "Interface is non interactive, so we are "
-                                "reraising: %s" % exc_str(e))
+                                "re-raising: %s" % exc_str(e))
```

Comment:
I think reraising was fine.

Reply:
ok, will change throughout the codebase... i.e. in 1 more place (a comment ;-))